files/galaxy/tpv/tools.yml

```diff
@@ -446,7 +458,7 @@ tools:
   toolshed.g2.bx.psu.edu/repos/galaxyp/diann/diann/.*:
     inherits: basic_docker_tool
   toolshed.g2.bx.psu.edu/repos/bgruening/instagraal/instagraal/.*:
-    inherits: basic_docker_tool
+    inherits: basic_pxe_gpu_tool
```

Comment:
Now I think  we will end up with two time --gpus on the commandlone I think. Can someone check if docker complains when a argument is given twice.